src/main/scala/cromwell/util/TryUtil.scala

```diff
@@ -69,19 +76,24 @@ object TryUtil {
       case Failure(f) if isFatal(f) => Failure(new CromwellFatalException(f))
       case value if (retryLimit.isDefined && retryLimit.get > 1) || retryLimit.isEmpty =>
         logFailures(value)
-        val retryCountMessage = if (retryLimit.getOrElse(0) > 0) s" (${retryLimit.getOrElse(0) - 1} more retries) " else ""
+
+        val transient = isTransient(value.failed.get)
+        val newLimit = if (transient) retryLimit else retryLimit.map(_ - 1)
+        val retryCountMessage = {
+          val transientMessage = if (transient) " - This error is transient and does not count against the retry limit." else ""
+          if (retryLimit.getOrElse(0) > 0 || transient) s" (${newLimit.getOrElse(0)} more retries)$transientMessage" else ""
+        }
+        val pollingInterval = backoff.backoffMillis
         val retryMessage = s"Retrying in $pollingInterval$retryCountMessage..."
         failMessage foreach { m => logger.warn(s"$m.  $retryMessage") }
 
-        Thread.sleep(pollingInterval.toMillis)
+        Thread.sleep(pollingInterval)
```

Comment:
I never paid attention to this before. This "smells" funny, but I'm not looking for any changes right now. I feel like to be a better async citizen, this logic should be using more akka, and scheduling some message to be sent again to some actor.